src/content/docs/en/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,529 @@
+
+See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for full release notes.
+
+### Removed: Support for Node 16
+
+Node 16 is scheduled to reach its End of Life in September 2023. 
+
+Astro v3.0 drops Node 16 support entirely, so that all Astro users can take advantage of Node's more modern features.
+
+#### What should I do?
+
+ Check that both your development environment and your deployment environment are using **Node `18.14.1` or later**.
+
+1. Check your local version of Node using:
+
+    ```sh
+    node -v
+    ```
+
+    If your local development environment needs upgrading, [install Node](https://nodejs.org/en/download/).
+
+
+2. Check your [deployment environment's](/en/guides/deploy/) own documentation to verify that they support Node 18. 
+
+    You can specify Node `18.14.1` for your Astro project either in a dashboard configuration setting, or a `.nvmrc` file.
+
+### Removed: Support for TypeScript 4
+
+In Astro v2.x, the `tsconfig.json` presets include support for both TypeScript 4.x and 5.x.
+
+Astro v3.0 updates the `tsconfig.json` presets to only support TypeScript 5.x. Astro now assumes that you use TypeScript 5.0 (March 2023), or that your editor includes it (e.g. VS Code 1.77).
+
+#### What should I do?
+
+If you have installed TypeScript locally, update to at least v5.0.
+
+```bash
+npm install typescript@latest --save-dev
+```
+
+### Removed: `@astrojs/image`
+
+In Astro v2.x, Astro offered an official image integration that included Astro `<Image />` and `<Picture />` components.
+
+Astro v3.0 removes this integration from the codebase entirely. Astro's new solution for images is a built-in image services API: `astro:assets`.
+
+#### What should I do?
+
+Remove the `@astrojs/image` integration from your project. You will need to not only uninstall the integration, but also update or remove any import statements and existing `<Image />` and `<Picture />` components. You might also need to configure a preferred default image processing service.
+
+You will find [complete, step-by-step instructions for removing the old image integration](/en/guides/images/#remove-astrojsimage) in our Images guide.
+
+Migrating to `astro:assets` will also bring some new image options and features that you may now wish to use. Please see the full [v3.0 Image Upgrade Advice](/en/guides/images/#upgrade-to-v30-from-v2x) for full details!
+
+```js del={3,7}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import image from '@astrojs/image';
+
+export default defineConfig({
+  integrations: [
+    image(),
+  ]
+})
+```
+
+### Removed: `<Markdown />` component
+
+In Astro v2.x, Astro deprecated the `<Markdown />` component and moved it to an external package.
```

Comment:
I think I made a mistake in my entry. The Markdown component is deprecated since Astro v1
```suggestion
In Astro v1.x, Astro deprecated the `<Markdown />` component and moved it to an external package.
```